main.go

```diff
@@ -8,7 +8,20 @@ import (
 	"github.com/deis/object-storage-cli/config"
 )
 
+var (
+	debugStrs = []string{"DEIS_DEBUG"}
+)
+
 func main() {
+	var debug bool
+	for _, str := range debugStrs {
```

Comment:
Fixed in https://github.com/deis/object-storage-cli/pull/12/commits/9dd8eb8fc819efc3e6e2fafe32084f42b49d8026